src/px-script/main/tabs/index.test.js

```diff
@@ -8,8 +8,8 @@ describe("px-script: tabs", () => {
     const div = document.createElement("div");
     const items = ["item1", "item2", "item3"];
 
-    const NoActiveTab = () => (
-        <div className="tabs">
+    const NoActiveTab = ({ id, tabsOpen }) => (
+        <div className={`tabs ${tabsOpen ? "tabs-open" : null}`} id={id}>
```

Comment:
Pretty sure that if `tabsOpen` is `falsey` the value `null` will remove the entire prop `className`
```suggestion
        <div className={`tabs${tabsOpen ? " tabs-open" : ""}`} id={id}>
```

Reply:
The tests work so maybe the "null" gets added as a string?

Reply:
In the tests (snapshots) i believe this will be `className={null}`, but i might be wrong.